models/ht_neuron.cpp

```diff
@@ -618,21 +661,27 @@ ht_neuron::update( Time const& origin, const long_t from, const long_t to )
         S_.y_ );              // neuron state
 
       if ( status != GSL_SUCCESS )
+      {
         throw GSLSolverFailure( get_name(), status );
+      }
     }
 
     // Deactivate potassium current after spike time have expired
     if ( S_.r_potassium_ && --S_.r_potassium_ == 0 )
+    {
       S_.g_spike_ = false; // Deactivate potassium current.
+    }
 
     // Add new spikes to node state array
     for ( size_t i = 0; i < B_.spike_inputs_.size(); ++i )
+    {
       S_.y_[ 2 + 2 * i ] +=
         V_.cond_steps_[ i ] * B_.spike_inputs_[ i ].get_value( lag );
+    }
```

Comment:
Another possible improvement: in the calibrate function there is a TODO:
In the original model there is a TODO (line 577, ht_neuron.cpp):    ```
// NOTE: code below initializes conductance step size for incoming pulses.
 // Variable and function names need to be changed!
```
Since you are already working on the ht_model, could your improve this?